eZ/Publish/API/Repository/Tests/FieldType/BaseIntegrationTest.php

```diff
@@ -923,7 +923,7 @@ public function testCopiedExternalData(Field $field)
      */
```

Comment:
One line above it is another `@dep_ends`

Reply:
> One line above it is another `@dep_ends`

Ok, removed all of them then via 53ead296a.